.github/workflows/bump.yaml

```diff
@@ -0,0 +1,18 @@
+name: Automatic version updates
+
+on:
+  schedule:
+    - cron: '0 9 * * *'
+  workflow_dispatch:
```

Comment:
Please make `workflow_dispatch` the first option to be consistent with other workflows